package.json

```diff
@@ -13,7 +13,7 @@
     "test": "turbo run test",
     "format": "prettier --write \"**/*.{ts,tsx,md}\"",
     "changeset": "changeset",
-    "version": "changeset version && npm install && cd packages/mystmd-py && bash scripts/bump-version.sh",
+    "version": "changeset version && npm install",
```

Comment:
Oh cool, this is nice.

Reply:
To be clear here, the version is only updated at pypi deployment time (because it's taken from package.json, which is copied in the deploy bash scipt).